### ISSUE TITLE
[0.76.0] Detected a flipper crash

### ISSUE THREAD

**Issue by jayshah123** [facebook/flipper#1944]
## 🐛 Bug Report

Flipper desktop app connected to android emulator experienced a crash in 

## To Reproduce

Flipper connected to react native debug app.
Device logs tab was selected in flipper desktop ui.

## Environment

Using flipper desktop (0.76.0) with android emulator Pixel_4_api_29
```
Error: Reducers may not dispatch actions.

Component stack:

    at Sheet (file:///private/var/folders/rb/njzjk7b13wlg_zd47d1rsrls1fp7y6/T/AppTranslocation/42052C91-EE3E-4FC7-B27F-65EFEA5DAE36/d/Flipper.app/Contents/Resources/app.asar/bundle.js:4747:1369)
    at ConnectFunction (file:///private/var/folders/rb/njzjk7b13wlg_zd47d1rsrls1fp7y6/T/AppTranslocation/42052C91-EE3E-4FC7-B27F-65EFEA5DAE36/d/Flipper.app/Contents/Resources/app.asar/bundle.js:36:976)
    at SheetRenderer (file:///private/var/folders/rb/njzjk7b13wlg_zd47d1rsrls1fp7y6/T/AppTranslocation/42052C91-EE3E-4FC7-B27F-65EFEA5DAE36/d/Flipper.app/Contents/Resources/app.asar/bundle.js:4739:126)
    at div
    at file:///private/var/folders/rb/njzjk7b13wlg_zd47d1rsrls1fp7y6/T/AppTranslocation/42052C91-EE3E-4FC7-B27F-65EFEA5DAE36/d/Flipper.app/Contents/Resources/app.asar/bundle.js:16:913
    at Layout.Top (file:///private/var/folders/rb/njzjk7b13wlg_zd47d1rsrls1fp7y6/T/AppTranslocation/42052C91-EE3E-4FC7-B27F-65EFEA5DAE36/d/Flipper.app/Contents/Resources/app.asar/bundle.js:1339:1754)
    at SandyApp (file:///private/var/folders/rb/njzjk7b13wlg_zd47d1rsrls1fp7y6/T/AppTranslocation/42052C91-EE3E-4FC7-B27F-65EFEA5DAE36/d/Flipper.app/Contents/Resources/app.asar/bundle.js:67:145)
    at div
    at file:///private/var/folders/rb/njzjk7b13wlg_zd47d1rsrls1fp7y6/T/AppTranslocation/42052C91-EE3E-4FC7-B27F-65EFEA5DAE36/d/Flipper.app/Contents/Resources/app.asar/bundle.js:16:913
    at ContextMenuProvider (file:///private/var/folders/rb/njzjk7b13wlg_zd47d1rsrls1fp7y6/T/AppTranslocation/42052C91-EE3E-4FC7-B27F-65EFEA5DAE36/d/Flipper.app/Contents/Resources/app.asar/bundle.js:62:1195)
    at PopoverProvider (file:///private/var/folders/rb/njzjk7b13wlg_zd47d1rsrls1fp7y6/T/AppTranslocation/42052C91-EE3E-4FC7-B27F-65EFEA5DAE36/d/Flipper.app/Contents/Resources/app.asar/bundle.js:4718:132)
    at TooltipProvider (file:///private/var/folders/rb/njzjk7b13wlg_zd47d1rsrls1fp7y6/T/AppTranslocation/42052C91-EE3E-4FC7-B27F-65EFEA5DAE36/d/Flipper.app/Contents/Resources/app.asar/bundle.js:3018:3008)
    at Provider (file:///private/var/folders/rb/njzjk7b13wlg_zd47d1rsrls1fp7y6/T/AppTranslocation/42052C91-EE3E-4FC7-B27F-65EFEA5DAE36/d/Flipper.app/Contents/Resources/app.asar/bundle.js:29:917)
    at AppFrame (file:///private/var/folders/rb/njzjk7b13wlg_zd47d1rsrls1fp7y6/T/AppTranslocation/42052C91-EE3E-4FC7-B27F-65EFEA5DAE36/d/Flipper.app/Contents/Resources/app.asar/bundle.js:3:941)

Error stacktrace:
Error: Reducers may not dispatch actions.
    at dispatch (file:///private/var/folders/rb/njzjk7b13wlg_zd47d1rsrls1fp7y6/T/AppTranslocation/42052C91-EE3E-4FC7-B27F-65EFEA5DAE36/d/Flipper.app/Contents/Resources/app.asar/bundle.js:42:4421)
    at Object.onHideSheet (file:///private/var/folders/rb/njzjk7b13wlg_zd47d1rsrls1fp7y6/T/AppTranslocation/42052C91-EE3E-4FC7-B27F-65EFEA5DAE36/d/Flipper.app/Contents/Resources/app.asar/bundle.js:42:6082)
    at Sheet.componentWillUnmount (file:///private/var/folders/rb/njzjk7b13wlg_zd47d1rsrls1fp7y6/T/AppTranslocation/42052C91-EE3E-4FC7-B27F-65EFEA5DAE36/d/Flipper.app/Contents/Resources/app.asar/bundle.js:4747:1775)
    at bj (file:///private/var/folders/rb/njzjk7b13wlg_zd47d1rsrls1fp7y6/T/AppTranslocation/42052C91-EE3E-4FC7-B27F-65EFEA5DAE36/d/Flipper.app/Contents/Resources/app.asar/bundle.js:59:87586)
    at cj (file:///private/var/folders/rb/njzjk7b13wlg_zd47d1rsrls1fp7y6/T/AppTranslocation/42052C91-EE3E-4FC7-B27F-65EFEA5DAE36/d/Flipper.app/Contents/Resources/app.asar/bundle.js:59:89385)
    at dk (file:///private/var/folders/rb/njzjk7b13wlg_zd47d1rsrls1fp7y6/T/AppTranslocation/42052C91-EE3E-4FC7-B27F-65EFEA5DAE36/d/Flipper.app/Contents/Resources/app.asar/bundle.js:59:101438)
    at e.unstable_runWithPriority (file:///private/var/folders/rb/njzjk7b13wlg_zd47d1rsrls1fp7y6/T/AppTranslocation/42052C91-EE3E-4FC7-B27F-65EFEA5DAE36/d/Flipper.app/Contents/Resources/app.asar/bundle.js:61:3794)
    at gg (file:///private/var/folders/rb/njzjk7b13wlg_zd47d1rsrls1fp7y6/T/AppTranslocation/42052C91-EE3E-4FC7-B27F-65EFEA5DAE36/d/Flipper.app/Contents/Resources/app.asar/bundle.js:59:44689)
    at Uj (file:///private/var/folders/rb/njzjk7b13wlg_zd47d1rsrls1fp7y6/T/AppTranslocation/42052C91-EE3E-4FC7-B27F-65EFEA5DAE36/d/Flipper.app/Contents/Resources/app.asar/bundle.js:59:99399)
    at Lj (file:///private/var/folders/rb/njzjk7b13wlg_zd47d1rsrls1fp7y6/T/AppTranslocation/42052C91-EE3E-4FC7-B27F-65EFEA5DAE36/d/Flipper.app/Contents/Resources/app.asar/bundle.js:59:95514)
    at file:///private/var/folders/rb/njzjk7b13wlg_zd47d1rsrls1fp7y6/T/AppTranslocation/42052C91-EE3E-4FC7-B27F-65EFEA5DAE36/d/Flipper.app/Contents/Resources/app.asar/bundle.js:59:44912
    at e.unstable_runWithPriority (file:///private/var/folders/rb/njzjk7b13wlg_zd47d1rsrls1fp7y6/T/AppTranslocation/42052C91-EE3E-4FC7-B27F-65EFEA5DAE36/d/Flipper.app/Contents/Resources/app.asar/bundle.js:61:3794)
    at gg (file:///private/var/folders/rb/njzjk7b13wlg_zd47d1rsrls1fp7y6/T/AppTranslocation/42052C91-EE3E-4FC7-B27F-65EFEA5DAE36/d/Flipper.app/Contents/Resources/app.asar/bundle.js:59:44689)
    at jg (file:///private/var/folders/rb/njzjk7b13wlg_zd47d1rsrls1fp7y6/T/AppTranslocation/42052C91-EE3E-4FC7-B27F-65EFEA5DAE36/d/Flipper.app/Contents/Resources/app.asar/bundle.js:59:44857)
    at ig (file:///private/var/folders/rb/njzjk7b13wlg_zd47d1rsrls1fp7y6/T/AppTranslocation/42052C91-EE3E-4FC7-B27F-65EFEA5DAE36/d/Flipper.app/Contents/Resources/app.asar/bundle.js:59:44792)
    at Jg (file:///private/var/folders/rb/njzjk7b13wlg_zd47d1rsrls1fp7y6/T/AppTranslocation/42052C91-EE3E-4FC7-B27F-65EFEA5DAE36/d/Flipper.app/Contents/Resources/app.asar/bundle.js:59:92663)
    at Oh (file:///private/var/folders/rb/njzjk7b13wlg_zd47d1rsrls1fp7y6/T/AppTranslocation/42052C91-EE3E-4FC7-B27F-65EFEA5DAE36/d/Flipper.app/Contents/Resources/app.asar/bundle.js:59:64370)
    at file:///private/var/folders/rb/njzjk7b13wlg_zd47d1rsrls1fp7y6/T/AppTranslocation/42052C91-EE3E-4FC7-B27F-65EFEA5DAE36/d/Flipper.app/Contents/Resources/app.asar/bundle.js:1329:1037
    at Array.forEach (<anonymous>)
    at AtomValue.notifyChanged (file:///private/var/folders/rb/njzjk7b13wlg_zd47d1rsrls1fp7y6/T/AppTranslocation/42052C91-EE3E-4FC7-B27F-65EFEA5DAE36/d/Flipper.app/Contents/Resources/app.asar/bundle.js:1329:1026)
    at AtomValue.set (file:///private/var/folders/rb/njzjk7b13wlg_zd47d1rsrls1fp7y6/T/AppTranslocation/42052C91-EE3E-4FC7-B27F-65EFEA5DAE36/d/Flipper.app/Contents/Resources/app.asar/bundle.js:1329:922)
    at MetroDevice.disconnect (file:///private/var/folders/rb/njzjk7b13wlg_zd47d1rsrls1fp7y6/T/AppTranslocation/42052C91-EE3E-4FC7-B27F-65EFEA5DAE36/d/Flipper.app/Contents/Resources/app.asar/bundle.js:3377:2990)
    at MetroDevice.destroy (file:///private/var/folders/rb/njzjk7b13wlg_zd47d1rsrls1fp7y6/T/AppTranslocation/42052C91-EE3E-4FC7-B27F-65EFEA5DAE36/d/Flipper.app/Contents/Resources/app.asar/bundle.js:3377:3013)
    at file:///private/var/folders/rb/njzjk7b13wlg_zd47d1rsrls1fp7y6/T/AppTranslocation/42052C91-EE3E-4FC7-B27F-65EFEA5DAE36/d/Flipper.app/Contents/Resources/app.asar/bundle.js:4065:2274
    at Proxy.filter (<anonymous>)
    at file:///private/var/folders/rb/njzjk7b13wlg_zd47d1rsrls1fp7y6/T/AppTranslocation/42052C91-EE3E-4FC7-B27F-65EFEA5DAE36/d/Flipper.app/Contents/Resources/app.asar/bundle.js:4065:2243
    at _e194.W.produce (file:///private/var/folders/rb/njzjk7b13wlg_zd47d1rsrls1fp7y6/T/AppTranslocation/42052C91-EE3E-4FC7-B27F-65EFEA5DAE36/d/Flipper.app/Contents/Resources/app.asar/bundle.js:69:14330)
    at e.default (file:///private/var/folders/rb/njzjk7b13wlg_zd47d1rsrls1fp7y6/T/AppTranslocation/42052C91-EE3E-4FC7-B27F-65EFEA5DAE36/d/Flipper.app/Contents/Resources/app.asar/bundle.js:4065:2216)
    at file:///private/var/folders/rb/njzjk7b13wlg_zd47d1rsrls1fp7y6/T/AppTranslocation/42052C91-EE3E-4FC7-B27F-65EFEA5DAE36/d/Flipper.app/Contents/Resources/app.asar/bundle.js:4094:2253
    at combination (file:///private/var/folders/rb/njzjk7b13wlg_zd47d1rsrls1fp7y6/T/AppTranslocation/42052C91-EE3E-4FC7-B27F-65EFEA5DAE36/d/Flipper.app/Contents/Resources/app.asar/bundle.js:42:2287)
    at rootReducer (file:///private/var/folders/rb/njzjk7b13wlg_zd47d1rsrls1fp7y6/T/AppTranslocation/42052C91-EE3E-4FC7-B27F-65EFEA5DAE36/d/Flipper.app/Contents/Resources/app.asar/bundle.js:4139:482)
    at Object.dispatch (file:///private/var/folders/rb/njzjk7b13wlg_zd47d1rsrls1fp7y6/T/AppTranslocation/42052C91-EE3E-4FC7-B27F-65EFEA5DAE36/d/Flipper.app/Contents/Resources/app.asar/bundle.js:42:4480)
    at unregisterDevices (file:///private/var/folders/rb/njzjk7b13wlg_zd47d1rsrls1fp7y6/T/AppTranslocation/42052C91-EE3E-4FC7-B27F-65EFEA5DAE36/d/Flipper.app/Contents/Resources/app.asar/bundle.js:4782:1453)
    at WebSocket.n.onclose.n.onerror (file:///private/var/folders/rb/njzjk7b13wlg_zd47d1rsrls1fp7y6/T/AppTranslocation/42052C91-EE3E-4FC7-B27F-65EFEA5DAE36/d/Flipper.app/Contents/Resources/app.asar/bundle.js:4782:1512)
```

**Comment by mweststrate** [comment]
Thanks for the report! We've been able to reproduce the issue and will land a fix in Flipper 0.78

**Comment by mweststrate** [comment]
0.78 has been released, closing
